### DISCUSSION TITLE
How to show the data as ABIF format?

### DISCUSSION THREAD

**Discussion by masiarek** [electorama/abif#34] [General]
Ranks
   G,B,P,R,O
18:1,5,4,2,3
12:5,1,4,3,2
10:5,2,1,4,3
9: 5,4,2,1,3
4: 5,2,4,3,1
2: 5,4,2,3,1

Ranks converted into Scores
   G,B,P,R,O
18:5,1,2,4,3
12:1,5,2,3,4
10:1,4,5,2,3
9: 1,2,4,5,3
4: 1,4,2,3,5
2: 1,2,4,3,5

Context:
forum - https://www.votingtheory.org/forum/topic/416/voting-example-pbs-different-methods-different-winners
docs: https://docs.google.com/document/d/1icQZ1efJV4XX7fD0_OTjNnW7uhfFV4lxbz5afTmqcyg/edit

**Comment by robla** [comment]
Your example is in this format:
```
Ranks
G,B,P,R,O
18:1,5,4,2,3
12:5,1,4,3,2
10:5,2,1,4,3
9: 5,4,2,1,3
4: 5,2,4,3,1
2: 5,4,2,3,1
```

Is there any published software out there that uses this format for expressing ranks?  It wouldn't be hard to add this to [electorama/abiftool](/electorama/abiftool), but before doing that, I want to make sure I don't clutter the tool with formats that aren't in widespread use.

**Comment by robla** [comment]
Here's one way the rankings above would be expressed in ABIF:
```
18:G>O>R>P>B
12:B>O>R>P>G
10:P>B>O>R>G
9: R>P>O>B>G
4: O>B>R>P>G
2: O>P>R>B>G
```

This is compatible with many online tools (e.g. my old tool, [Rob LeGrand's tool](https://www.cs.angelo.edu/~rlegrand/rbvote/calc.html), etc.)
